src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1234,6 +1273,41 @@
 				}
 			};
 
+			prototype._overscrollTop = function () {
+				var self = this,
+					state = self._state,
+					scroll = state.scroll;
+
+				state.duration = OVERSCROLL_IN_DURATION;
+
+				// start scroll animation from current scroll position
+				scroll.from = scroll.current;
+				scroll.to = OVERSCROLL_TOP;
+
+				// if scroll animation is ended then animation start
+				if (self._scrollAnimationEnd) {
+					state.startTime = Date.now();
+					self._scrollAnimationEnd = false;
+					self._requestRender();
+				}
+
+				// clear timeout
+				if (overscrollRotaryHandler) {
```

Comment:
Wouldn't be better to have overscrollRotaryHandler as member of ArcListview?

Reply:
Done